platos_flood.py

```diff
@@ -0,0 +1,84 @@
+import load_json
+import numpy as np
+from math import floor
+
+# generate 3d matrix with sum of chair voxels in each cell
+def generate_average_voxel_matrix (resolution):
+
+    # load models
+    models = load_json.load_folder('models', resolution)
+
+    # empty 3d matrix of ints to hold value sums
+    voxel_matrix = np.zeros((resolution, resolution, resolution), np.int8)
+
+    # sum chair voxels into matrix
+    voxel_count = 0
+    for chair in models:
+        for vox in chair:
+            voxel_matrix[vox[0], vox[1], vox[2]] += 1
+            voxel_count += 1
+
+    return voxel_matrix, floor(voxel_count / len(models))
+
+def distance (coords, resolution):
+    half = floor(resolution / 2)
+    d = 0
+    for coord in coords:
+        d += abs(coord - half)
+
+    return -d
```

Comment:
i tried it with distance maximized (no - before return d) and its really cool!

![platos_chair_dist_max_0](https://user-images.githubusercontent.com/5488759/41966452-feb58ca2-79b3-11e8-9888-17001db58d15.png)
![platos_chair_dist_max_1](https://user-images.githubusercontent.com/5488759/41966459-03d88b30-79b4-11e8-96d9-85ce7b3cafaa.png)